en/components/splitter.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Angular Splitter | Split Panes |  Ignite UI for Angular | Infragistics
+_description: Use the Angular Splitter component to create a simple split layout splitting the view horizontally or vertically into multiple collapsible split panes.
+_keywords: angular splitter, igniteui for angular, infragistics
+---
+
+## Splitter
+
+The Ignite UI for Angular Splitter component provides a framework for a simple split layout splitting the view horizontally or vertically into multiple smaller areas called split panes.
+
+### Demo
+<div class="sample-container loading" style="height: 450px">
+    <iframe id="splitter-horizontal-iframe" src='{environment:dvDemosBaseUrl}/layouts/splitter-horizontal' width="100%" height="100%" seamless frameBorder="0" onload="onXPlatSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" disabled class="stackblitz-btn"   data-iframe-id="splitter-horizontal-iframe" data-demos-base-url="{environment:dvDemosBaseUrl}">View on StackBlitz
+    </button>
+</div>
+
+<div class="divider--half"></div>
+
+### Usage
+
+To start using the **igxSplitter** first we need to import the **IgxSplitterModule** in our **app.module**:
+```typescript
+// app.module.ts
+...
+import { IgxSplitterModule } from 'igniteui-angular';
+
+@NgModule({
+    ...
+    imports: [..., IgxSplitterModule],
+    ...
+})
+export class AppModule {}
+```
+
+After that we can add the markup for our component:
+```html
+<!-- splitter.component.html -->
+<igx-splitter>
+    <igx-splitter-pane>
+        ...
+    </igx-splitter-pane>
+    <igx-splitter-pane>
+        ...
+    </igx-splitter-pane>
+    <igx-splitter-pane>
+        ...
+    </igx-splitter-pane>
+</igx-splitter>
+```
+The Initialization of the **igxSplitter** is done by using the **igx-splitter** tag. Multiple splitter panes can be defined under a single **igx-splitter**. The content of the pane is templatable and will be rendered in its own resizable container.
+
+
+#### Orientation
+
+The splitter can be vertical or horizontal, which is defined by the [`type`]({environment:angularApiUrl}/classes/igxsplittercomponent.html#type) input. The default value is Vertical.
+
+```html
+<igx-splitter [type]="SplitterType.Vertical">
```

Comment:
Enums cannot be accessed directly from the template. This would throw error. Should be bound to a prop in the component:
<igx-splitter [type]="type">

...
public type = SplitterType.Horizontal;